client/state/selectors/is-requesting-jetpack-scan-history.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import 'state/data-layer/wpcom/sites/scan/history';
+
+/**
+ * Returns true if we are currently making a request to retrieve Jetpack Scan History. False otherwise.
+ * Returns null if the site is unknown, or there is no information yet.
+ *
+ * @param  {object}   state    Global state tree
+ * @param  {number}   siteId   The ID of the site we're querying
+ * @returns {?boolean}          Whether the connection data is being requested
+ */
+export default function isRequestingJetpackScanHistory( state, siteId ) {
+	return get( state.jetpackScan.history.requestStatus, [ siteId ], null ) === 'pending';
```

Comment:
Not seeing how this function could return a `null` value?

Reply:
Thanks for catching this. I made the selector return either false or true and fixed the docstring.